firmware/quadruna/BMS/src/io/ltc6813/io_ltc6813Shared.h

```diff
@@ -17,7 +17,7 @@ typedef enum
     ACCUMULATOR_SEGMENT_1,
     ACCUMULATOR_SEGMENT_2,
     ACCUMULATOR_SEGMENT_3,
-    ACCUMULATOR_SEGMENT_4,
+    // ACCUMULATOR_SEGMENT_4,
     ACCUMULATOR_NUM_SEGMENTS,
```

Comment:
it appears making the num segments is bricking some other code in the build. please check that out.